Kickstarter-iOS/Views/ShimmerLoadingViews/PledgeShippingLocationShimmerLoadingView.swift

```diff
@@ -0,0 +1,74 @@
+import Foundation
+import Library
+import Prelude
+import UIKit
+
+final class PledgeShippingLocationShimmerLoadingView: UIView, ShimmerLoading {
+  // MARK: - Properties
+
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var buttonPlaceholder: UIView = {
+    UIView(frame: .zero)
+      |> \.shimmersWhenLoading .~ true
+  }()
+
+  private lazy var amountPlaceholder: UIView = {
+    UIView(frame: .zero)
+      |> \.shimmersWhenLoading .~ true
+  }()
+
+  // MARK: - Lifecycle
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+
+    self.configureViews()
+    self.setupConstraints()
+
+    self.startLoading()
+  }
+
+  required init?(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  override func layoutSubviews() {
+    super.layoutSubviews()
+
+    self.layoutGradientLayers()
```

Comment:
It's always going to be necessary to call this on the `ShimmerLoading`-conforming view, right?

Reply:
Hmm, yeah because we can't override `layoutSubviews` in an extension 🤔